pytorch_lightning/callbacks/progress.py

```diff
@@ -253,6 +254,17 @@ def disable(self) -> None:
     def enable(self) -> None:
         self._enabled = True
 
+    def get_progress_bar_dict(self, trainer):
+        metrics_holder = trainer.logger_connector._progress_bar_metrics
+        metrics_holder._to_float = True
+        model_ref = trainer.get_model()
+        metrics_holder.convert(
+            self.trainer._device_type == DeviceType.TPU,
+            model_ref.device if model_ref is not None else model_ref
+        )
```

Comment:
```suggestion
        metrics_holder.convert(self.trainer._device_type == DeviceType.TPU, model_ref.device)
```
trainer.get_model() is never None right?

Reply:
It is None before the first .fit() or .test() call happens :)

Reply:
I meant during training.. it can't be None in the code above.